### PR TITLE
Creating steering functionality (actually this time)

### PR DESCRIPTION
**Pull request by RobertM3**
please accept it

### CODE REVIEW

File: src/main/java/com/lincolnRobotics/lincolnSimulation/SimulationRobotMotion.java [hunk 1]
```diff
@@ -143,10 +189,12 @@ public double getLoopHertz() {
      * the robot model controlled by this robot motion
      */
     private RobotModel robotModel;
+    private double robotWidth =  50;
```
[the remark this answers is not in the document]
Comment (pearlbob) [reply src/main/java/com/lincolnRobotics/lincolnSimulation/SimulationRobotMotion.java]: correct.  we'll fix the pull request.

[the remark this answers is not in the document]
Comment (LonelyKirbyMain) [reply src/main/java/com/lincolnRobotics/lincolnSimulation/SimulationRobotMotion.java]: you can do`robotWidth = robotModel.getWidthCm();` or delete

File: src/main/java/com/lincolnRobotics/lincolnSimulation/SimulationRobotMotion.java [hunk 2]
```diff
@@ -118,6 +124,44 @@ void applyTankMotionTick() {
                 robotModel.getX(), robotModel.getY(), robotModel.getRotation()));
     }
 
+    void applySteeringMotionTick() {
```
Comment (LonelyKirbyMain) [inline src/main/java/com/lincolnRobotics/lincolnSimulation/SimulationRobotMotion.java]: you don't take the robot's width into account

**Comment by RobertM3** [comment]
Damaged pull req

**Pull request closed without merging**